### ISSUE TITLE
Store User Flight History

### ISSUE THREAD

**Issue by gageayk** [American-Capstones/TAMU-Spring-2022#15]
### Pre-requisites

- [X] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find an existing bug.

### Is your feature request related to a problem? Please describe.
Once the user's flight history has been retrieved, the information should be temporarily stored in the database so that any querying and filtering does not need to occur multiple times. 

### Describe the solution you'd like
- Store data in the database for temporary use
- Create the framework for filtering

### Describe alternatives you've considered
Instead of storing the data, the query could just be rerun. But if a user wants to rerun the recommendations algorithm, then the query and any filtering done on the flights will have to be done again.

### Additional context
- The framework for filtering should just be a function or section within the code prior to storage. The actual code for filtering will be created later in another issue
- This issue is dependent on #14
